Topic 0/src/point2/Architect.java

```diff
@@ -0,0 +1,39 @@
+package point2;
+
+import java.util.ArrayList;
+
+
+public class Architect {
+
+	private Builder roombuilder;
+	private ArrayList<Room> rooms; //room list includes the architect to build the house
+
```

Comment:
[Medium] This room list could be part of *Builder*, if the builder builds a *Building* instead of *Room*